Gruntfile.js

```diff
@@ -1,203 +1,217 @@
-          'src/jquery.js',
-
-          // End the closure
-          'src/outro.js'
-        ],
-      }
-    },
-    uglify: {
-      dist: {
-        src: 'dist/jsoneditor.js',
-        dest: 'dist/jsoneditor.min.js'
-      },
-      options: {
-        preserveComments: 'some',
-        sourceMap: true
-      }
-    },
-    watch: {
-      scripts: {
-        files: ["src/**/*.js"],
-        tasks: ["concat"]
-      }
-    },
-    jshint: {
-      options: {
-        browser: true,
-        indent: 2,
-        devel:true,
-        nonbsp: true,
-        nonew: true,
-        immed: true,
-        latedef: true
-      },
-      beforeconcat: [
-        'src/class.js',
-        'src/ie9.js',
-
-        // Utils like extend, each, and trigger
-        'src/utilities.js',
-
-        // The main JSONEditor class
-        'src/core.js',
-
-        // JSON Schema validator
-        'src/validator.js',
-
-        // All the editors
-        'src/editor.js',
-        'src/editors/*.js',
-
-        // All the themes and iconlibs
-        'src/theme.js',
-        'src/themes/*.js',
-        'src/iconlib.js',
-        'src/iconlibs/*.js',
-
-        // The JS templating engines
-        'src/templates/*.js',
-
-        // Set the defaults
-        'src/defaults.js',
-
-        // Wrapper for $.fn style initialization
-        'src/jquery.js'
-      ],
-      afterconcat: {
-        options: {
-          undef: true
-        },
-        files: {
-          src: ['dist/jsoneditor.js']
-        }
-      }
-    },
-    connect: {
-        default: {
-            options: {
-                port: 9000,
-                hostname: '0.0.0.0',
-                debug: true,
-                keepalive: true
-            }
-        },
-        testing: {
-            options: {
-                port: 9001,
-                hostname: '0.0.0.0',
-                debug: true,
-                keepalive: true
-            }
-        }
-    },
-    run: {
-        options: {
-            // Task-specific options
-        },
-        mocha: {
-            cmd: 'mocha',
-            args: [
-                'tests/selenium/*.js',
-                '--reporter=nyan'
-            ]
-        }
-    }
-  });
-
-  // These plugins provide necessary tasks.
-  grunt.loadNpmTasks('grunt-string-replace')
-  grunt.loadNpmTasks('grunt-contrib-uglify');
-  grunt.loadNpmTasks('grunt-contrib-watch');
-  grunt.loadNpmTasks('grunt-contrib-jshint');
-  grunt.loadNpmTasks('grunt-contrib-concat');
-  grunt.loadNpmTasks('grunt-contrib-connect');
-  grunt.loadNpmTasks('grunt-run');
-
-  // Serve files
-  grunt.registerTask('serve', 'connect:default');
-  grunt.registerTask('serve-test', 'connect:testing');
-
-  // Run mocha tests
-  grunt.registerTask('test', ['run:mocha']);
-
-    // Default task.
-  grunt.registerTask('default', ['jshint:beforeconcat','concat','jshint:afterconcat','uglify']);
-  
-  grunt.registerTask('rawbuild', ['concat','uglify']);
-  
-};
+'use strict';

```

Comment:
Can you double check the `Gruntfile` changes, look like mostly whitespace to me.

Reply:
I just added code for minifying CSS files from src/ to dist/ using "grunt-contrib-cssmin" <https://github.com/gruntjs/grunt-contrib-cssmin#readme> 

I normally use Gulp, so I'm not that familiar with Grunt plugins.  😄